.github/workflows/ci.yml

```
@@ -10,16 +10,17 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
-      - name: Setup Node.js
-        uses: actions/setup-node@b39b52d1213e96004bfcb1c61a8a6fa8ab84f3e8 # v4.0.1
+      - name: Install pnpm
+        uses: pnpm/action-setup@d882d12c64e032187b2edb46d3a0d003b7a43598 # v2.4.0
+      - uses: actions/setup-node@v4.0.1
         with:
-          node-version: 20
-          cache: npm
+          cache: "pnpm"
+          node-version-file: ".node-version"
       - name: Install dependencies
-        run: npm install
+        run: pnpm install --frozen-lockfile --strict-peer-dependencies
```

--strict-peer-dependenciesは.npmrcにあるので要らなそう
```suggestion
        run: pnpm install --frozen-lockfile
```